src/ProductConstructionService/ProductConstructionService.LongestBuildPathUpdater/LongestBuildPathUpdater.cs

```diff
@@ -27,7 +27,7 @@ public LongestBuildPathUpdater(
     public async Task UpdateLongestBuildPathAsync()
     {
         List<Channel> channels = [.. _context.Channels.Select(c => new Channel() { Id = c.Id, Name = c.Name })];
-        IReadOnlyList<string> frequencies = new[] { "everyWeek", "twiceDaily", "everyDay", "everyBuild", "none", };
+        IReadOnlyList<string> frequencies = new[] { "everyWeek", "twiceDaily", "everyDay", "everyBuild", "none", "everyTwoWeeks", "everyMonth" };
```

Comment:
considering the list is used here too, maybe we should move the list from `Darc.Constants` do DarcLib so we have it only in one place